pipelines/skylab/optimus/Optimus.wdl

```diff
@@ -93,6 +93,7 @@ workflow Optimus {
   String star_docker = "star:1.0.1-2.7.11a-1692706072"
```

Comment:
Sorry for the drive-by comment after the PR was merged:

This declaration appears to be no longer used after the changes in this PR (this was caught by our CI process for `sprocket` where we analyze public WDL repositories for changes to diagnostics emitted by sprocket).